schema/src/lib.rs

```diff
@@ -47,7 +47,7 @@ pub trait IntoSchema {
 /// Applicable for types that represents decimal place of fixed point
```

Comment:
inside `schema/src/bin/main.rs` we generate derive schemas for our structures. Would you be able to add a test there that would ensure no field is `usize`/`isize`?

Reply:
if you go with serde fields I mentioned in the other comment, you may have to modify `IntoSchema` macro

Reply:
Good suggestion. I will take a look at this tomorrow.